### PR TITLE
DTSPB1036-Welsh AccessbilityTests

### PR DESCRIPTION
**Pull request by phaniperla**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-1036

Caveat FE Welsh Accessibility Tests

### Change description ###

Caveat FE Welsh Accessibility Tests

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Review by arun-hmcts — APPROVED** [review]

**Review by vinceholt — APPROVED** [review]

File: bin/run-nightly-functional-tests.sh [hunk 1]
```diff
@@ -1,21 +1,7 @@
 #!/bin/bash
 set -ex
 
+TEST_URL='https://probate.aat.platform.hmcts.net/caveats/start-apply'
```
Comment (Rob-Doherty) [inline bin/run-nightly-functional-tests.sh:4]: Is this needed if the URL is passed in by the Jenkins_nightly file? by hardcoding it here, it defeats the purpose of making it manually configurable

File: Jenkinsfile_nightly [hunk 2]
```diff
@@ -4,7 +4,7 @@ properties([
         pipelineTriggers([cron('00 20 * * *')]),
         parameters([
                 string(name: 'ENVIRONMENT', defaultValue: 'aat', description: 'Environment to test'),
-                string(name: 'URL_TO_TEST', defaultValue: 'https://probate.aat.platform.hmcts.net', description: 'The URL you want to run these tests against'),
+                string(name: 'URL_TO_TEST', defaultValue: 'https://probate.aat.platform.hmcts.net/caveats/start-apply', description: 'The URL you want to run these tests against'),
```
Comment (Rob-Doherty) [inline Jenkinsfile_nightly:7]: will setting the URL here to a specific page break the functional tests? do the functional tests require a base URL to add specific page endings to, like in other repos?

Comment (phaniperla) [reply Jenkinsfile_nightly:7]: it won't break the functional tests . We have two shell scripts 1) run-functional-tests.sh  2)  run-nightly-functional-tests.sh
 run-nightly-functional-tests.sh --> it calls "test:nightly" script  to run e2e tests marked with tag "@nightly" so i don't see any issue there. Let's have a call?

File: bin/run-nightly-functional-tests.sh (hunk 1 above)
Comment (phaniperla) [reply bin/run-nightly-functional-tests.sh:4]: let's have call regarding this?

Comment (phaniperla) [reply bin/run-nightly-functional-tests.sh:4]: Jenkins nightly pipeline passing https://probate.aat.platform.hmcts.net/' and it's not passing above URL

**Review by Rob-Doherty — APPROVED** [review]
👍

**Pull request merged**